getting-started/background-processing/TranslateUI/Startup.cs

```diff
@@ -42,7 +42,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<PublisherClient>(
                 provider => PublisherClient.CreateAsync(new TopicName(
                     GetProjectId(), GetTopicName())).Result);
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+            services.AddMvc(options => options.EnableEndpointRouting = false).SetCompatibilityVersion(CompatibilityVersion.Version_2_1);

```

Comment:
I'd personally split this before the ".SetCompatibilityVersion"

Reply:
Yep, done